src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.1.2
+++++++
+* Updated the help text for the --node-osdisk-size parameter.
```

Comment:
From the PR checklist, do not add history entries for help text changes. This should just say "Minor fixes."

Reply:
Thanks. I have tried to change it but Travis is not happy. I have no idea what I have done? :-)